charts/fees-register-api/Chart.yaml

```diff
@@ -1,9 +1,13 @@
-apiVersion: v1
+apiVersion: v2
 appVersion: "1.0"
 description: Helm chart for the HMCTS fees-register api
 name: fees-register-api
 home: https://github.com/hmcts/ccfr-fees-register-app
-version: 0.3.0
+version: 0.3.2
 maintainers:
   - name: HMCTS Fees & Payments Dev Team
     email: ccpay@hmcts.net
+dependencies:
+  - name: java
+    version: ~3.0.0-beta
```

Comment:
don't use a beta version if you don't have to this is well out of date:
https://github.com/hmcts/chart-java/releases

Reply:
The version has been upgraded now